app/views/affiliates/index.html.erb

```diff
@@ -1,6 +1,13 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Affiliates</h1>
+<div class="row index-header">
+  <div class="col-md-8 title">
+    <span>Affiliates</span>
```

Comment:
Before:
![screen shot 2017-03-26 at 01 20 45](https://cloud.githubusercontent.com/assets/15064985/24328842/b8200a74-11c2-11e7-8c14-f2246a54a90b.png)
After:
![screen shot 2017-03-26 at 01 20 58](https://cloud.githubusercontent.com/assets/15064985/24328843/c06207e6-11c2-11e7-959e-891ed04dbfd5.png)


Emmm, why bother to change the size? We should just use rails magic by calling `<h2> ... </h2>` if you want to simply change the size.